lib/CfTemplateGenerators/Sns.test.js

```diff
@@ -50,5 +50,48 @@ describe('Sns', () => {
       const actual = Sns.replaceTopicSubscriptionFunctionWithAlias(snsTopic, functionAlias, functionName);
       expect(actual).to.deep.equal(expected);
     });
+
+    it("replaces the sns subscription's function for an alias", () => {
```

Comment:
This test case is wrapped into the `describe('.replaceTopicSubscriptionFunctionWithAlias', ...)` block, even though is testing a different function (`.replaceSubscriptionFunctionWithAlias`). That should be changed

Reply:
changed to wrapped in different function test case